client/blocks/login/continue-as-user.jsx

```diff
@@ -1,4 +1,5 @@
-import { Button } from '@automattic/components';
+import { Button as A8CButton } from '@automattic/components';
```

Comment:
Typescript tells me this button has been marked as deprecated so I've chosen to make `Button` the WordPress components button, as that's what we should be using from now on.